examples/libvirt.yaml

```diff
@@ -11,10 +11,11 @@ admin:
 baseDomain:
 
 libvirt:
-  # You must specify an IP address here that libvirtd is listening on,
-  # and that the cluster-api controller pod will be able to connect
-  # to.  Often 192.168.122.1 is the default for the virbr0 interface.
-  uri: qemu+tcp://192.168.122.1/system
+  uri: qemu:///system
```

Comment:
Without the IP address, how does the cluster API controller talk to `libvirtd`?

Reply:
See below. It's talking via the gateway / bridge now.